### PR TITLE
Techstyle companion

### PR DESCRIPTION
**Pull request by isa-leroux448**
🎟️ Ticket(s): Closes #

👷 Changes: A brief summary of what changes were introduced.
- looooots of style changes to fit techstyle theme
- removing/commenting out any blueprint content (e.g. redemption messages)
- ~~add zoom on tap option for carousel in companion~~
- change redemption success animation to dress
- ~~added in google form embed after design carousel so users can vote directly~~

💭 Notes: Any additional things to take into consideration.

STILL NEED TO DO:
- add attending companies
- add mentors

Wait! Before you merge, have you checked the following:

📷 Screenshots
Demo (1)

## Checklist

- [X] Looks good on large screens
- [X] Looks good on mobile

https://github.com/ubc-biztech/bt-web/assets/106799810/9c3f865f-77ac-480f-b9c0-19ab67c9df0b

### CODE REVIEW

File: src/pages/public/Companion/events/index.js [hunk 1]
```diff
@@ -8,8 +8,64 @@ import MISLogo from "../../../../assets/2024/misnight/logo.svg";
...
+import TechStyleLogo from "../../../../assets/2024/techstyle/techstyle_logo.png";
+import TechStyleTexture from "../../../../assets/2024/techstyle/techstyle_grid.png";
+import TSLogo from "../../../../assets/2024/techstyle/techstyle_biztech_logo.png";
 
 export default [
+  {
+    /* Date indicating when event ends, this also indicates which app the companion app will render, to be safe, but a couple days after event ends */
+    activeUntil: new Date(new Date("03-06-2024").getTime() + (7 * 24 * 60 * 60 * 1000)), // change 3 days after date
+    /* id of event in dynamodb, used for queries */
+    eventID: "blueprint",
```
Comment (isa-leroux448) [inline src/pages/public/Companion/events/index.js:21]: need to change this to techstyle

File: src/pages/public/Companion/components/ShowcaseCard.js [hunk 2]
```diff
@@ -64,33 +76,51 @@ function ShowcaseCard({
...
+          ><CloseIcon style={{color: "black"}}/></button>
+        </>
+        :
+        <Box className={classes.container} style={{
+          backgroundColor: "white",
+        // backgroundImage: `url(card${(pos % 2) + 1}.svg)`
+        }}>
+          <img
+            alt={"project-picture"}
+            src={Test}
```
Comment (isa-leroux448) [inline src/pages/public/Companion/components/ShowcaseCard.js:102]: need to change this back to {img} (and one above as well)

File: src/pages/public/Companion/events/TechStyle2024.js [hunk 3]
```diff
@@ -0,0 +1,305 @@
...
+} from "@trendyol-js/react-carousel";
+import Sched from "../../../../assets/2024/techstyle/techstyle_schedule.png";
+import Mentors from "../components/mentor/Mentors";
+import Podium from "../components/Podium";
+import GamificationActivityTable from "../components/GamificationActivityTable";
+import GamificationRewardTable from "../components/GamificationRewardTable";
+import ShowcaseCard from "../components/ShowcaseCard";
+import FeedbackForm from "../components/FeedbackForm";
+
+const VotingLink = "https://docs.google.com/forms/d/e/1FAIpQLSeLmzRHnr48NHsY_rgXlvJOsbxu-X1ehAWuSFrDyOw7WwKqPg/viewform?usp=sf_link";
```
Comment (isa-leroux448) [inline src/pages/public/Companion/events/TechStyle2024.js:33]: will need to replace with actual voting link

**Comment by ddennis924** [comment]
can u publish the pr

**Comment by isa-leroux448** [comment]
I published but still need to update points info and fix failing checks

**Review by jerryxu99 — APPROVED** [review]
Looks sooo good 
Great job conforming to the codebase and also thanks for catching all the Blueprint specific award messages

File: src/pages/public/Companion/components/CompanionLayout.js [hunk 4]
```diff
@@ -325,7 +327,7 @@ const CompanionLayout = (params) => {
   const [input, setInput] = useState("");
   const [isModalOpen, setIsModalOpen] = useState(false);
   const [transition, setShowTransition] = useState(true);
```
Comment (ddennis924) [inline src/pages/public/Companion/components/CompanionLayout.js:329]: I think we need to set transition as false too

File: src/pages/public/Companion/events/TechStyle2024.js [hunk 5]
```diff
@@ -0,0 +1,240 @@
...
+  };
+});
+
+const activities = [
+  {
+    name: "Early check-in",
+    points: "25 points"
+  },
+  {
+    name: "Scan a Linkedin QR Code on a professional's name tag (can only redeem once)",
```
Comment (ddennis924) [inline src/pages/public/Companion/events/TechStyle2024.js:51]: need to remove this one, cant do this anymore cuz no qr codes on nametags

File: src/pages/public/Companion/components/CompanionLayout.js (hunk 4 above)
Comment (ddennis924) [inline src/pages/public/Companion/components/CompanionLayout.js:329]:
```suggestion
  const [transition, setShowTransition] = useState(false);
```

**Pull request merged**